roc_example_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat_test
+
+import (
+	"fmt"
+
+	"github.com/gonum/stat"
+)
+
+func ExampleROC() {
+	y := []float64{0, 3, 5, 6, 7.5, 8}
+	classes := []bool{true, false, true, false, false, false}
+	weights := []float64{4, 1, 6, 3, 2, 2}
+
+	tpr, fpr := stat.ROC(0, y, classes, weights)
+	fmt.Printf("true  positive rate: %v\n", tpr)
+	fmt.Printf("false positive rate: %v\n", fpr)
+
+	// Output:
+	// true  positive rate: [0 0.4 0.4 1 1 1 1]
+	// false positive rate: [0 0 0.125 0.125 0.5 0.75 1]
+}
+
+func ExampleROC_withoutWeights() {
```

Comment:
How does this render on godoc? I would think that the default case would be this one and the ExampleROC_Weighted would be the qualified one (or even ExampleROC_Unweighted and ExampleROC_Weighted).